web/themes/custom/recipe_theme/recipe_theme.libraries.yml

```diff
@@ -8,3 +8,8 @@ style:
   dependencies:
     - core/jquery
     - core/drupal
+
+fonts:
+  css:
+    theme:
+      '//fonts.googleapis.com/css2?family=Dosis&display=swap': { external: true }
```

Comment:
Please download the fonts to your codebase. So it won't depend on an external source.